### ISSUE TITLE
RHEL8 installer requires perl-interpreter, not perl

### ISSUE THREAD

**Issue by tomuxi** [dotless-de/vagrant-vbguest#368]
Since "perl" package is not available in the RHEL8 repositories the vagrant-vbguest checkup fails when upping a RHEL8 box. RHEL8 would need perl-interpreter instead of perl.

**Comment by skipperTux** [comment]
Looks like the 'stale bot' is slightly overzealous in tagging/closing issues ;) This is still an issue with RHEL 8. As @tomuxi said vbguest needs to check for a different perl-package-name on RHEL 8, see sample output below.
To check for Red Hat OS version there are [multiple options](https://www.cyberciti.biz/faq/how-do-i-determine-rhel-version/), like `$ cat /etc/redhat-release  # e.g. Red Hat Enterprise Linux release 8.2 (Ootpa)`.


```
==> rhel8: Machine booted and ready!                                                   
[rhel8] GuestAdditions versions on your host (6.1.14) and guest (5.2.44) do not match. 
Extra Packages for Enterprise Linux 8 - x86_64   39 kB/s |  29 kB     00:00            
Extra Packages for Enterprise Linux 8 - x86_64  3.7 MB/s | 8.0 MB     00:02            
Last metadata expiration check: 0:00:03 ago on Mon 05 Oct 2020 03:26:03 PM UTC.        
Package kernel-devel-4.18.0-193.el8.x86_64 is already installed.                       
Package kernel-devel-4.18.0-193.el8.x86_64 is already installed.                       
Package gcc-8.3.1-5.el8.x86_64 is already installed.                                   
Package binutils-2.30-73.el8.x86_64 is already installed.                              
Package make-1:4.2.1-10.el8.x86_64 is already installed.                               
No match for argument: perl                                                            
Package bzip2-1.0.6-26.el8.x86_64 is already installed.                                
Package elfutils-libelf-devel-0.178-7.el8.x86_64 is already installed.                 
Error: Unable to find a match: perl                                                    

[vagrant@rhel8 ~]$ sudo dnf install perl                                  
Failed to set locale, defaulting to C.UTF-8                               
Last metadata expiration check: 0:01:07 ago on Mon Oct  5 15:41:53 2020.  
No match for argument: perl                                               
Error: Unable to find a match: perl                                       
[vagrant@rhel8 ~]$ sudo dnf install perl-interpreter                      
Failed to set locale, defaulting to C.UTF-8                               
Last metadata expiration check: 0:01:12 ago on Mon Oct  5 15:41:53 2020.  
Package perl-interpreter-4:5.26.3-416.el8.x86_64 is already installed.    
Dependencies resolved.                                                    
Nothing to do.                                                            
Complete!                                                                 
```

**Comment by fnordfish** [comment]
Is there a quick way to figure out which of those packages is available? Like `yum list perl-interpreter` or something?
I like to avoid version checks at  (almost) all costs. They just get out of date so fast :)
Also, the Red Had installer is not only used on RHEL, but basically everything that looks like it (except centos).

**Comment by skipperTux** [comment]
`perl-interpreter` or `perl` are the actual package names, so something like `<packagemanager> list perl-interpreter` will look for the package. [It seems complex](https://unix.stackexchange.com/questions/453460/why-doesnt-rpm-or-dnf-recognize-default-perl) to check for the availability of perl. Here are the responses from some rpm-based distros using vagrant and [generic images](https://app.vagrantup.com/generic):

**CentOS 7**
```
[vagrant@centos7 ~]$ yum list perl                                                                                     
[...]
Installed Packages                                                                                                     
perl.x86_64                                           4:5.16.3-295.el7                                            @base
[vagrant@centos7 ~]$ yum list perl-interpreter                                                                         
[...]
Error: No matching Packages to list                                                                                    
```

**CentOS 8**
```
[vagrant@centos8 ~]$ yum list perl                                                                                     
[...]
Installed Packages                                                                                                     
perl.x86_64                                         4:5.26.3-416.el8                                         @AppStream
[vagrant@centos8 ~]$ yum list perl-interpreter                                                                         
[...]
Installed Packages                                                                                                     
perl-interpreter.x86_64                                    4:5.26.3-416.el8                                     @BaseOS
```
See RHEL 8 remark about yum/dnf.

**RHEL 7**
```
[vagrant@rhel7 ~]$ yum list perl                                                                                       
[...]
Installed Packages                                                                                                     
perl.x86_64                                     4:5.16.3-295.el7                                     @InstallMedia     
Available Packages                                                                                                     
perl.x86_64                                     4:5.16.3-297.el7                                     rhel-7-server-rpms
[vagrant@rhel7 ~]$ yum list perl-interpreter                                                                           
[...]
Error: No matching Packages to list                                                                                    
```

**RHEL 8**
```
[vagrant@rhel8 ~]$ yum list perl                                                                                       
[...]
Error: No matching Packages to list                                                                                    
[vagrant@rhel8 ~]$ yum list perl-interpreter                                                                           
[...]
Installed Packages                                                                                                     
perl-interpreter.x86_64                               4:5.26.3-416.el8                                @rhel8-base-media

```
Remark: `yum` still works, however [`dnf` is the new package manager](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/considerations_in_adopting_rhel_8/index#notable-changes-to-the-yum-stack_software-management).

**Fedora 32**
(my machine, no vagrant)
```
$ yum list perl
bash: yum: command not found...

$ dnf list perl
[...]
Installed Packages
perl.x86_64                              4:5.30.3-456.fc32                              @updates
$ dnf list perl-interpreter
[...]
Installed Packages
perl-interpreter.x86_64                        4:5.30.3-456.fc32                        @updates
```

**Comment by fnordfish** [comment]
Fixed in v0.26.0
